ui/src/common/utils.js

```diff
@@ -143,3 +143,8 @@ export function wait(milisec) {
     setTimeout(() => resolve(), milisec);
   });
 }
+
+// appending `s` is all that's needed for the current usages
```

Comment:
I would still use something that would work for future cases as well so that we don't come back and change this behavior. Why not use the pluralize mentioned in the US?

Reply:
Because it's unnecessarily complex (multiple regexp matches etc, + few 100 LOC that you need to ship for no reason) and only useful when you're dealing with dynamic values, not in our case where words are static (hardcoded).

When we have a case that doesn't work we can add map `singularToPlural` and use that instead of that library.